### PR TITLE
[v7r1] RSS DB schema change to include a VO column

### PR DESCRIPTION
**Pull request by martynia**
Add a VO column to Resource Status and Resource Management DB tables. 

BEGINRELEASENOTES
*ResourceStatusSystem
CHANGE: add a VO column to all tables as apart of the primary key and a default value='all'

ENDRELEASENOTES
TODO:
- [ ] we'll include the required schema updates for existing instances in the release notes

### CODE REVIEW

File: tests/Integration/ResourceStatusSystem/Test_ResourceStatus.py [hunk 1]
```diff
@@ -128,7 +129,7 @@ def test_addAndRemove(self):
     self.assertTrue(res['OK'])
     self.assertEqual(res['Value'][0][0], 'TestName1234')
     self.assertEqual(res['Value'][0][1], 'statusType')
-    self.assertEqual(res['Value'][0][2], 'Banned')
+    self.assertEqual(res['Value'][0][3], 'Banned')
     print("inserted lastCheckTime and DateEffective: %s, %s" % (res['Value'][0][7], res['Value'][0][4]))
```
Comment (andresailer) [inline tests/Integration/ResourceStatusSystem/Test_ResourceStatus.py:133]: you also need to increment these indices (7 and 4), also in other places.

Comment (martynia) [reply tests/Integration/ResourceStatusSystem/Test_ResourceStatus.py]: Thank you for spotting this. Done.

**Comment by atsareg** [comment]
How urgent is this PR. We try not to make DB schema changes in a patch PRs but rather in the new releases.

**Comment by martynia** [comment]
It is a base for future changes to go in, so it would be covenient to have it early. However, if you find this problematic to go in a patch it can go in the new release.

**Comment by fstagni** [comment]
Given that:
- the timeline of v7r2 is ATM uncertain
- I am not sure anyway is already in production with v7r1
- RSS is not used by every installation

I would keep v7r1 as a target here.

**Comment by atsareg** [comment]
OK, let's merge it. This is exactly the case where an automatic schema evolution would be very useful

**Review by atsareg — APPROVED** [review]

**Pull request merged**